classes/02class/exercises/c02-network02/hsy3418/c02-network02.txt

```diff
@@ -0,0 +1,26 @@
+
+Given a AWS network range, 
+slice it in 4 small subnets and identify how many usable hosts for each one of them.
+
+---
+10.0.0.0/16
+total available addresses: 2 pow 16 = 65536
+65536 /4 = 16384 addresses
+So each subnet could have 16384 addresses, usable hosts = 16384-5 = 16379
+
+subnet1 range: 10.0.0.4 - 10.0.63.254
```

Comment:
how many hosts per subnet?

Reply:
it's been calculated in the description(16379), maybe it's not very clear, just made some changes.

Reply:
you also need to add what is the netmask for each of your subnets.. the number after the /